client/lib/signup/dependency-store.js

```diff
@@ -1,50 +1,38 @@
 /**
  * External dependencies
  */
-var store = require( 'store' ),
-	assign = require( 'lodash/assign' ),
-	keys = require( 'lodash/keys' ),
-	difference = require( 'lodash/difference' ),
-	isEmpty = require( 'lodash/isEmpty' );
+import keys from 'lodash/keys';
+import difference from 'lodash/difference';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
This is awesome! Initially I wanted to do it this way, but was rejected by the ESlint rules.